tfk8s.go

```diff
@@ -88,6 +88,12 @@ func snakify(s string) string {
 	return strings.ToLower(re.ReplaceAllString(s, "_"))
 }
 
+// escape incidences of ${} with $${} to prevent Terraform trying to interpolate them
+func escapeShellVars(s string) string {
+	r := regexp.MustCompile(`(\${.*?})`)
```

Comment:
The `.*?` here is to make the regexp not greedy – otherwise it would match `${VAR1} ${VAR2}` as one group.